src/components/AccountsList/index.js

```diff
@@ -24,30 +26,53 @@ function AccountsList({
   const handleClose = () => setOpen(false);
   const [listAccounts, setListAccounts] = useState(accounts);
   const [newAccounts, setNewAccounts] = useState("");
-  const fetcher = (url) => fetchJson(url);
-  const { data, mutate } = useSWR(apiUrl, fetcher);
+  const [existingAccounts, setExistingAccounts] = useState(accounts);
+  const [page, setPage] = useState();
```

Comment:
@gertie-sheshe this shouldn't be undefined.. or your initial call to fetch data will fail. So either you pass a props or use default of 1 (i.e first page)
```suggestion
  const [page, setPage] = useState(pageProp || 1);
```